### PR TITLE
Fix typos and turn the Tournament Schedule into a table in the MWC4K 2018 news post

### PR DESCRIPTION
**Pull request by HappyStick**
---

### CODE REVIEW

File: news/2018-07-25-MWC4K-2018-registrations-open.md [hunk 1]
```diff
@@ -13,19 +13,22 @@ You can discuss this event and follow the most important changes in the **[offic
...
-+ Registration Phase: 2018-07-25/2018-08-08
-+ Drawings: 2018-08-18 (14:00 UTC)
-+ Group Stage: 2018-08-25/2018-08-26
-+ Round of 16: 2018-09-01/2018-09-02
-+ Quarterfinals: 2018-09-08/2018-09-09
-+ Semifinals: 2018-09-15/2018-09-16
-+ Finals: 2018-09-22/2018-09-23
-+ Grand Finals: 2018-09-29/2018-09-30
+ 
+| Phase              | Dates                   |
```
Comment (TPGPL) [inline news/2018-07-25-MWC4K-2018-registrations-open.md]: Phase -> Event
Dates -> Timestamp

File: news/2018-07-25-MWC4K-2018-registrations-open.md [hunk 2]
```diff
@@ -13,19 +13,22 @@ You can discuss this event and follow the most important changes in the **[offic
...
+| ------------------ | ----------------------- |
+| Registration       | July 25th - Aug 8th     |
```
Comment (TPGPL) [inline news/2018-07-25-MWC4K-2018-registrations-open.md]: Registration Phase

File: news/2018-07-25-MWC4K-2018-registrations-open.md [hunk 3]
```diff
@@ -13,19 +13,22 @@ You can discuss this event and follow the most important changes in the **[offic
...
+| Drawings           | Aug 18th                |
```
Comment (TPGPL) [inline news/2018-07-25-MWC4K-2018-registrations-open.md]: Live Drawings

File: news/2018-07-25-MWC4K-2018-registrations-open.md [hunk 4]
```diff
@@ -13,19 +13,22 @@ You can discuss this event and follow the most important changes in the **[offic
...
+| Group Stage        | Aug 25th & 26th         |
+| Round of 16        | Sept 1st & 2nd          |
+| Quarterfinals      | Sept 8th & 9th          |
+| Semifinals         | Sept 15th & 16th        |
+| Finals             | Sept 22nd & 23rd        |
+| Grand Finals       | Sept 29th or 30th       |
```
Comment (TPGPL) [inline news/2018-07-25-MWC4K-2018-registrations-open.md]: Could you return the YYYY-MM-DD dates?

File: news/2018-07-25-MWC4K-2018-registrations-open.md [hunk 5]
```diff
@@ -13,19 +13,22 @@ You can discuss this event and follow the most important changes in the **[offic
...
  
 ## Prizes
-+ **1st:** $150 per team member, profile badge, "osu!mania Champion" user title for one year
-+ **2nd:** $80 per team member, Profile badge
-+ **3rd:** $40 per team member, Profile badge
+**1st:** $150 per team member, profile badge, "osu!mania Champion" user title for one year  
+**2nd:** $80 per team member, Profile badge  
+**3rd:** $40 per team member, Profile badge
```
Comment (TPGPL) [inline news/2018-07-25-MWC4K-2018-registrations-open.md]: Delete extra spaces after lines 29-31

File: news/2018-07-25-MWC4K-2018-registrations-open.md [hunk 6]
```diff
@@ -13,19 +13,22 @@ You can discuss this event and follow the most important changes in the **[offic
...
  
-+ Registration Phase: 2018-07-25/2018-08-08
-+ Drawings: 2018-08-18 (14:00 UTC)
-+ Group Stage: 2018-08-25/2018-08-26
-+ Round of 16: 2018-09-01/2018-09-02
-+ Quarterfinals: 2018-09-08/2018-09-09
-+ Semifinals: 2018-09-15/2018-09-16
-+ Finals: 2018-09-22/2018-09-23
-+ Grand Finals: 2018-09-29/2018-09-30
+
```
Comment (Joehuu) [inline news/2018-07-25-MWC4K-2018-registrations-open.md]: Remove this line.

File: news/2018-07-25-MWC4K-2018-registrations-open.md (hunk 5 above)
Comment (Joehuu) [inline news/2018-07-25-MWC4K-2018-registrations-open.md]: Remove this extra line.

**Pull request merged**